tests/js/session-tests.js

```diff
@@ -461,6 +461,25 @@ module.exports = {
         });
     },
 
+    testSyncV1() {
```

Comment:
Perhaps this in not the norm for the tests, but I would recommend a comment on what this test does, so I don't have to check the code.
"SyncV1" doesn't tell me enough.

Reply:
If you want to expose that in the naming: "testThrowsWhenOpeningIncompatibleRealms" might do.